src/components/hoc/Card.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Button from "../hoc/Button";
+import { formatPrice } from "../../utils/maths";
+import styled from "styled-components";
+import { theme } from "../../theme/theme";
+
+export default function Card({ key, title, imageSource, price }) {
+  return (
+    <CardStyled key={key} className="card">
```

Comment:
```suggestion
    <CardStyled className="card">
```
le key doit toujours être à l'extérieur du composant pour que React sache s'il faut le conserver ou le détruire pour le remplacer par une autre instance de Card

Reply:
Correction apportée